libs/librepcb/libraryeditor/pkg/fsm/packageeditorstate_select.cpp

```diff
@@ -230,6 +230,20 @@ bool PackageEditorState_Select::processGraphicsSceneRightMouseButtonReleased(
   }
 }
 
+bool PackageEditorState_Select::processSelectAll() noexcept {
+  switch (mState) {
+    case SubState::IDLE: {
+      if (auto item = mContext.currentGraphicsItem) {
+        item->setSelectionRect(
+            item->boundingRect().adjusted(-100, -100, 100, 100));
```

Comment:
Are these the max bounds, or just "magic numbers that should cover everything"?

Reply:
They are "magic numbers that should cover everything" :slightly_smiling_face: But note it takes the actual bounding rect and increases it a bit just to be sure everything is really *inside* that rect. I first wrote a `selectAll()` method which does really select all items, but somehow I was not happy with adding many lines of code for something which could be solved with that simple hack :see_no_evil:

Reply:
Ahh, I missed the expansion part. Makes sense :slightly_smiling_face:

Reply:
I added a comment to clarify it :)